make_wininsts.bat

```diff
@@ -1,10 +1,10 @@
-SET PATH=C:\Python26;%PATH%
```

Comment:
What happened to this file, the rest of the `*.bat`, and `st.cmd` ?  I assume there is line ending change, was that intentional?